data/net/minecraft/world/inventory/Slot.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/world/inventory/Slot
+	METHOD <init> (Lnet/minecraft/world/Container;III)V
+		ARG 1 container
+		ARG 2 slotNumber
```

Comment:
The field for this is just called `slot` (which is kinda terrible). I would suggest calling this `containerIndex`, as it is the index in the `Container`. Otherwise just call it `slot`, but not `slotNumber`.